tests/flow/tests_pytorch.py

```diff
@@ -495,7 +448,7 @@ def test_pytorch_scriptexecute_errors(env):
         check_error_message(env, con, "CROSSSLOT Keys in request don't hash to the same slot", 'AI.SCRIPTEXECUTE', 'ket{1}', 'bar', 'KEYS', 1 , '{2}', 'INPUTS', 2, 'a{1}', 'b{1}', 'OUTPUTS', 1, 'c{1}')
 
         # key doesn't exist
-        check_error_message(env, con, "tensor key is empty or in a different shard", 'AI.SCRIPTEXECUTE', 'ket{1}', 'bar', 'KEYS', 1 , '{1}', 'INPUTS', 2, 'a{1}', 'b{2}', 'OUTPUTS', 1, 'c{1}')
+        check_error_message(env, con, "CROSSSLOT Keys in request don't hash to the same slot", 'AI.SCRIPTEXECUTE', 'ket{1}', 'bar', 'KEYS', 1 , '{1}', 'INPUTS', 2, 'a{1}', 'b{2}', 'OUTPUTS', 1, 'c{1}')
```

Comment:
Why does the error changed?

Reply:
since OSS Redis uses `getkeys-api` and we mark `INPUTS` scope in script commands as a scope that contains keys. so in our case `a{1}` and `b{2}` are not mapped to the same shared.
This moved from RedisAI error to Redis error on OSS Redis usage.